src/cascadia/TerminalApp/GlobalAppSettings.cpp

```diff
@@ -173,6 +173,7 @@ void GlobalAppSettings::LayerJson(const Json::Value& json)
             warnings = winrt::TerminalApp::implementation::Command::LayerJson(_commands, bindings);
             // It's possible that the user provided commands have some warnings
             // in them, similar to the keybindings.
+            _keybindingsWarnings.insert(_keybindingsWarnings.end(), warnings.begin(), warnings.end());

```

Comment:
please check `git blame` and the history for this file -- i have a feeling that we might have deleted this line on purpose.

Reply:
Good catch. This ended up being a stray comment from https://github.com/microsoft/terminal/issues/6949.

Removing the comment.